code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -136,7 +137,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 			equipped = TRUE
 
 /obj/item/pda/proc/update_label()
-	name = "PDA-[owner] ([ownjob])" //Name generalisation
+	name = "[owner] ([ownjob])" //Name generalisation

```

Comment:
This causes problems with the way verb parsing works in BYOND, where in some cases clicks will flip between "Joe Bine" the human and "Joe Bine (Captain)" the PDA. It should be left as-is. See #40988.

Reply:
Wow, that's... Wow

Reply:
Allright, no longer changes the item name and just reads off owner info instead.